source/css/_common/scaffolding/base.styl

```diff
@@ -27,17 +27,11 @@ h1, h2, h3, h4, h5, h6 {
 for headline in (1..6) {
   h{headline} {
     font-size: $font-size-headings-base - $font-size-headings-step * headline;
-    code {
-      font-size: 1em;
```

Comment:
Maybe better vice versa: to use `em` instead of this?

Reply:
`code {font-size: 1em}` is duplicate. Already exist in `normalize.styl`